defaults/main.yml

```diff
@@ -10,44 +10,42 @@ omero_ms_image_region_folder: '/opt/omero/microservice'
 # Microservice port
 omero_ms_image_region_port: '8081'
 
-# Omero Database URL
+# OMERO Database URL
 omero_ms_image_region_db_url: '127.0.0.1'
 
-# Omero database server port
+# OMERO database server port
 omero_ms_image_region_db_port: '5432'
 
-# Omero database name
+# OMERO database name
 omero_ms_image_region_db_name: 'omero'
 
-# Omero database user name
+# OMERO database user name
 omero_ms_image_region_db_username: 'omero'
 
-# Omero database user password
+# OMERO database user password
 omero_ms_image_region_db_pass: 'omero'
 
-# Log level, it can be info, debug or error
+#Logging level, allowed values: `info`, `debug`,  `error`
 omero_ms_image_region_log_level: 'info'
 
-# Omero data folder
+# OMERO data folder
 omero_data_dir: '/OMERO'
 
-# Omero scripts folder
+# OMERO scripts folder
 omero_script_repo_root: '/opt/omero/server/OMERO.server/lib/scripts/'
 
-# No of Microservice workers, the default is double the number of processors which the machine has
+# No of the Microservice workers, the default is double the number of processors which the machine has
```

Comment:
No -> Number (if you use abbreviation No., you must use the dot as well, otherwise it means negation)